index.html

```diff
@@ -17,7 +21,7 @@
 <body>
   <header>
     <nav id="cabecalho">
-      <img id="logo_pqn" src="./assets/img/logos-soberana/logo-com-texto.png" alt="Logo" href="./index.html" width="160" height="160" />
+      <img id="logo_pqn" src="./assets/img/logos-soberana/logo-com-texto.png" alt="Logo da Soberana" href="./index.html" width="160" height="160" />

```

Comment:
```suggestion
      <img id="logo_pqn" src="./assets/img/logos-soberana/logo-com-texto.png" alt="Logo da Soberana TV" href="./index.html" width="160" height="160" />

```
melhora a sintaxe pro leitor de tela (acessibilidade) e tambem impacta no SEO do google pra aparecer tambem se buscar "sobreana tv" separado.